modules/markdown.php

```diff
@@ -13,30 +13,3 @@
  */
 
 include dirname( __FILE__ ) . '/markdown/easy-markdown.php';
-
-/**
- * Markdown should be enabled for posts whenever the module is active.
- * We don't need any additional checkbox in Settings > Writing for that option.
- *
- * In addition to the filter above, the option is also automatically set when
- * the Markdown module is activated, thanks to the jetpack_activate_module_markdown filter.
- *
- * @see https://github.com/Automattic/jetpack/pull/6548
- */
-
-// Step 1. Force the option on whenever this file is loaded (i.e. when the module is acive).
-add_filter( 'pre_option_' . WPCom_Markdown::POST_OPTION, '__return_true' );
-
-/**
```

Comment:
Yes, I had re-added that function in d5280531cb2eb276377e9cbabb61d05bd05f54c0, and it was removed in c54efbd85df32b349bd6e30f2c315c186254c70c. I think we should keep it.